modules/ROOT/pages/rest-sdk/sample-data.adoc

```diff
@@ -0,0 +1,141 @@
+= Define a Sample Data Resolver
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+Sample Data Resolvers provide an example of how an operation's output looks like. It is the main insight of an operation's output, as it not only shows the data structure, but it shows a real example of that structure and sample values for each field in the structure.
+
+Sample Data Resolvers can be applied to operations (paged or not) and triggers.
```

Comment:
Can we add a screenshot of how sample data looks like in Composer? Many users are not familiar with this concept as it's new. See slide #4 https://docs.google.com/presentation/d/1ksH8LWxF3FkyGSChBFtLuFv2CgepTiDaXUpZxfNeI3k/edit#slide=id.gb417afadab_0_52